src/Core/Content/Media/SalesChannel/MediaRoute.php

```diff
@@ -35,7 +35,12 @@ public function getDecorated(): AbstractMediaRoute
         throw new DecorationPatternException(self::class);
     }
 
-    #[Route(path: '/store-api/media', name: 'store-api.media.detail', methods: ['POST'])]
+    #[Route(
+        path: '/store-api/media',
+        name: 'store-api.media.detail',
+        methods: ['POST', 'GET'],
+        defaults: ['_httpCache' => true]
```

Comment:
would be good chance to use some constants instead of strings 😁 

```suggestion
        methods: [\Symfony\Component\HttpFoundation\Request::METHOD_POST, \Symfony\Component\HttpFoundation\Request::METHOD_GET],
        defaults: ['\Shopware\Core\PlatformRequest::ATTRIBUTE_HTTP_CACHE' => true]
```

Reply:
and sorting the parameter of the Route attribute. some of them have a different sorting

Reply:
Changed. Should we also replace `PlatformRequest::ATTRIBUTE_ENTITY => 'product'` with `PlatformRequest::ATTRIBUTE_ENTITY => ProductDefinition::ENTITY_NAME` and so on? 🤔

Reply:
if you like :+1: